tests/regression/02-base/60-malloc-nonrefl.c

```diff
@@ -0,0 +1,25 @@
+#include<stdlib.h>
+#include<stdint.h>
+
+void* malloc_2(size_t s) {
+    return malloc(s);
+}
+
+int main() {
+    int* ptr1 = malloc_2(sizeof(int));
+    int* ptr2 = malloc_2(sizeof(int));
+
+    // This is technically UB (comparing unrelated pointers)
+    assert(ptr1==ptr2); // UNKNOWN!
+
+    // CIL turn this into the following, that is not UB and will fail in the concrete
+    // Goblint claims that this holds
```

Comment:
Maybe update this comment?

Reply:
Are the other comments about UB still right? We discussed this in Munich but I don't remember what the conclusion was and this test probably wasn't updated.

Reply:
> Are the other comments about UB still right?

No, they are wrong. In fact, `==` and `!=` can be used to compare any pointers of compatible types. It is only other comparisons (`<`,`<=`, ...) that are undefined.